src/main/java/io/ix0rai/bodaciousberries/block/BasicBerryBush.java

```diff
@@ -233,7 +231,7 @@ public static ActionResult pickBerries(BlockPos pos, World world, BlockState sta
             if (currentAge == maxAge - 1 && giveRipeBerry) {
                 dropStack(world, pos, new ItemStack(berryType, berryAmount));
             }
-        } else {
+        } else if (currentAge == maxAge) {
```

Comment:
```suggestion
        } else {
```
why did I even change this